haas/templates/base.html

```diff
@@ -15,10 +15,25 @@
   <!--[if IE 8]>
     <link href="{{ url_for('static', filename='govuk-frontend-ie8-4.1.0.min.css') }}" rel="stylesheet">
   <![endif]-->
+
+  {{ super() }}
+
+  <!-- Google Tag Manager -->
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer','{{ container_id }}');</script>
+  <!-- End Google Tag Manager -->
+  
 {% endblock %}
 
 {% block bodyStart %}
   {% block gtm_body %}{% endblock %}
```

Comment:
This line isn't used - I think the tag you have added below was intended to go in here, but you could remove the `gtm_body` block altogether. I don't think there's any reason to put the GTM tag in a block unless you intend to replace it in derived templates.